app/src/main/java/protect/card_locker/MainActivity.java

```diff
@@ -685,6 +731,10 @@ private void toggleSelection(int inputPosition) {
             mCurrentActionMode.setTitle(getResources().getQuantityString(R.plurals.selectedCardCount, count, count));
 
             MenuItem editItem = mCurrentActionMode.getMenu().findItem(R.id.action_edit);
+
+            MenuItem archiveItem = mCurrentActionMode.getMenu().findItem(R.id.action_archive_unarchive);
+            archiveItem.setVisible(true);
```

Comment:
I think it makes more sense to turn these into 2 menu items. One for archiving, one for unarchiving.

That way you can:
1. Show only archiving if all items aren't archived
2. Show only unarchiving if all items are archived
3. Show both if the items are in a mixed state

That also prevents confusion with state flipping around.

Reply:
The thing is that i don't really get what do you want, i mean you have the checkbox in order to see the archived and all or only the unarchived ones

Reply:
The thing is that i don't really get what do you want, i mean you have the checkbox in order to see the archived and all or only the unarchived ones

Reply:
Oh yeah but i already implemented that the only thing if you select more than one you get the options Archive All and Unarchive all